src/components/WorkOrder/AssetTypeField.js

```diff
@@ -80,17 +80,23 @@ export default class AssetTypeField extends Component {
 
   componentDidMount() {
     this.setState({ loading: true });
-    let userPosition = {};
-    navigator.geolocation.getCurrentPosition(pos => {
-      userPosition["lat"] = pos.coords.latitude;
-      userPosition["lon"] = pos.coords.longitude;
 
-      getSignalsOptions(userPosition).then(data => {
-        this.setState({
-          signalOptions: data,
-          loading: false,
-          userPosition: userPosition,
-        });
+    let watchPosition = {};
+    navigator.geolocation.watchPosition(pos => {
+      watchPosition["lat"] = pos.coords.latitude;
+      watchPosition["lon"] = pos.coords.longitude;
+
+      getSignalsOptions(watchPosition).then(data => {
```

Comment:
let's try implementing a counter so that we never request the position more than 10 times. We can test how that works.

Reply:
Done and done. I went with a limit of 20 since I watched the counter increment again, and it updated a lot faster than I remembered. I'll roll the latest changes in a PR to merge to production at the end of the day and do some testing before tomorrow.